lib/typing/ext/variables.go

```diff
@@ -29,7 +29,6 @@ var supportedDateFormats = []string{
 	PostgresDateFormat,
 }
 
-// TODO: Remove
 var SupportedTimeFormatsLegacy = []string{
```

Comment:
Should we rename this to `SupportedTimeFormats` if we're going to keep it?

Reply:
Yup sgtm